app/assets/stylesheets/user.sass

```diff
@@ -12,7 +12,7 @@ section.user
       h2
         font-size: 16px
         font-weight: normal
-      button
+      .btn
```

Comment:
Bootstrap also has a really useful `pull-right` class that you can add to any element to float it right.